app/helpers/organisations_helper.rb

```diff
@@ -1,5 +1,13 @@
 module OrganisationsHelper
-  def stringify_organisations(organisations)
-    organisations.collect(&:title).join(', ').html_safe
+  def stringify_organisations(organisations, options = {})
```

Comment:
Have you thought about reusing this function every time we have to render the organisations of a Content Item?
The code would be more readable.